doc/Learn/Tutorials/Authentication/HowTo-Cookies.md

```diff
@@ -3,11 +3,88 @@ uid: Learn.Tutorials.Authentication.HowToCookieAuthorization
 ---
 # How-To: Using Cookies to Authorize
 
-`Uno.Extensions.Authentication` provides you with a consistent way to add authentication to your application. It is recommended to use one of the built in `IAuthenticationService` implementations. This tutorial will use the custom authorization to validate user credentials.
+An authorization method is conceptually similar to using the system credential store to use **cookies**. Cookies are a common way to store tokens that can be used to authenticate a user. When an HTTP request is successfully authenticated, the server will return a response that we can use to create a cookie. The cookie can then be used to authenticate future requests. Uno Extensions makes these cookie-related authorization steps less tedious by doing the work of extracting the token(s) from the authentication response, storing it in a cookie, and applying the cookie to future requests. This tutorial will teach you how to configure authentication to apply tokens from a cookie when they are available.
```

Comment:
I'm not sure about the wording of this paragraph. We don't use cookies for storing tokens, but where Cookie Authorization is used, we attach cookies that contain the token values to each service request